user/puzzles/template.py

```diff
@@ -0,0 +1,28 @@
+pkgname = "puzzles"
+pkgver = "20240802"
+pkgrel = 0
+build_style = "cmake"
+hostmakedepends = [
+    "cmake",
+    "halibut",
+    "imagemagick",
+    "ninja",
+    "perl",
+    "pkgconf"
+]
+makedepends = ["gtk+3-devel"]
+depends = ["cmd:xdg-open!xdg-utils"]
+pkgdesc = "Collection of small programs which implement one-player puzzle games"
+maintainer = "Earldridge Jazzed Pineda <earldridgejazzedpineda@gmail.com>"
+license = "MIT"
+url = "https://www.chiark.greenend.org.uk/~sgtatham/puzzles"
+source = "https://www.chiark.greenend.org.uk/~sgtatham/puzzles/puzzles.tar.gz"
+sha256 = "f72e9ea630011ba0acdf59b74fff24f0e4f33ae6271ff8312c5122b517d1c249"
+hardening = ["vis", "cfi"]
+# No tests
+options = ["!check"]
+# Compilation of icons has a tendency to fail if any directory specified has a
+# hyphen (-). The srcdir name unfortunately cannot be changed :(
```

Comment:
or rather you'll have to fix that somehow since otherwise it won't work

but from a quick look this isn't the reason it fails to build, it works after you retry it enough times and the crashes are something invalid passed to pango seemingly, so good luck :)

Reply:
actually it seems to not fail if you add a font to hostmakedeps, i.e. like `"fonts-dejavu-ttf"`

Reply:
but maybe i'm thinking of something different? what kind of 'tendency to fail' did you run into based on the name of the directory?

Reply:
That is also why the text is missing on generated icons.

Reply:
can confirm; just adding the font fixes it then :D

Reply:
What font can we use here?

Reply:
`"fonts-dejavu"`

Reply:
you can also remove this now; the issue all along was a missing font and the comment is also not needed anymore